sdk/apimanagement/Azure.ResourceManager.ApiManagement/samples/Generated/Samples/Sample_ApiManagementUserResource.cs

```diff
@@ -118,7 +118,7 @@ public async Task Update_ApiManagementUpdateUser()
         public async Task GenerateSsoUri_ApiManagementUserGenerateSsoUrl()
         {
             // Generated from example definition: specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-03-01-preview/examples/ApiManagementUserGenerateSsoUrl.json
-            // this example is just showing the usage of "User_GenerateSsoUri" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "User_GenerateSsoUrl" operation, for the dependent resources, they will have to be created separately.
```

Comment:
I think `Url` is the correct operation ID: https://github.com/Azure/azure-rest-api-specs/blob/c852a76b96e5c71972cdc62a303665fec7c867b6/specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2024-05-01/apimusers.json#L422